command/tls.go

```diff
@@ -0,0 +1,62 @@
+package command
+
+import (
+	"os"
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+type TLSCommand struct {
+	Meta
+}
+
+func FileDoesNotExist(file string) bool {
```

Comment:
This function is only used within the command package, so I think we can make this private.